delivery_carrier_label_batch/wizard/generate_labels.py

```diff
@@ -277,6 +277,12 @@ def _concat_files(self, file_type, files):
         if file_type == 'pdf':
             return assemble_pdf(files)
         if file_type == 'zpl2':
-            return assemble_zpl2(files)
+            zpl2_single_images = self.env['ir.config_parameter'].get_param(
```

Comment:
No need to call `safe_eval` on it? I can't see a call of it in https://github.com/odoo/odoo/blob/9.0/openerp/addons/base/ir/ir_config_parameter.py#L57-L75 so I suppose that even if `"False"` is set we are evaluating the value to true.